parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -131,7 +143,11 @@ write_files:
   content: |
     [Service]
     ExecStart=
+    {{if .IsCoreOS}}
+    ExecStart=/usr/bin/env PATH=${TORCX_BINDIR}:${PATH} ${TORCX_BINDIR}/dockerd --host=fd:// --containerd=/var/run/docker/libcontainerd/docker-containerd.sock --storage-driver=overlay2 --bip={{WrapAsParameter "dockerBridgeCidr"}} $DOCKER_SELINUX $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_BIP $DOCKER_OPT_MTU $DOCKER_OPT_IPMASQ
+    {{else}}
     ExecStart=/usr/bin/dockerd -H fd:// --storage-driver=overlay2 --bip={{WrapAsParameter "dockerBridgeCidr"}}
+	{{end}}
```

Comment:
nit: let's keep the existing whitespace/indentation usage pattern

Reply:
to clarify, anything that's not whitespace sensitive should be fully left-justified?

Reply:
I wasn't sure if the stuff inside `content` should be indented with the text misaligned to make it more pronounced

Reply:
Well, you're correct that the template layer tokens that open and close block/lexical scope (e.g., `{{`) aren't whitespace-sensitive, but for readability/maintainability we like to align them.

E.g.,

```
        {{if foo}}
bar baz...
        {{end}}
```

So in this case let's add 4 spaces to the `{{end}}` on L150 so it's left-aligned w/ the ` {{if .IsCoreOS}}`.

Reply:
Looks like you got it :)